test/integration/node-specific/feature_flags.test.ts

```diff
@@ -42,4 +45,35 @@ describe('Feature Flags', () => {
       });
     }
   });
+
+  describe('@@mdb.enableMongoLogger', () => {
+    let cachedEnv;
+    const loggerFeatureFlag = Symbol.for('@@mdb.enableMongoLogger');
+
+    before(() => {
+      cachedEnv = process.env;
+      process.env['MONGODB_LOG_COMMAND'] = 'emergency';
```

Comment:
probably want to split this into two `context` blocks, one for when logging for a component is enabled, and one for when it is not enabled

Reply:
added!